lib/widgets/customized_text.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+
+class customizedText extends StatelessWidget {
```

Comment:
Bunun ismi için yukarıda yorum bırakmıştım. Ona göre güncelleyebilirsin

Reply:
Hocam gerekli düzenlemeyi yaptım CustomText olarak tanımladım.